docs/ref/settings.txt

```diff
@@ -2007,7 +2007,7 @@ Default: ``''`` (Empty string)
 
 A secret key for a particular Django installation. This is used to provide
 :doc:`cryptographic signing </topics/signing>`, and should be set to a unique,
-unpredictable value.
+unpredictable value and valid unicode string.
```

Comment:
what's an "invalid unicode string"?
value -> "Unicode string" (or whatever wording you decide on).